mitmproxy/web/app.py

```diff
@@ -176,6 +176,7 @@ def get(self):
         sio.close()
 
     def post(self):
+        # self.state.clear()
```

Comment:
what is the best way to stop mitmweb from recording, i think we also need to resume recording when a user clicks the menuitem 'new'